src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py

```diff
@@ -972,16 +972,29 @@ def customizePerJob(self, job):
 
         # Performance estimates
         if job.get('estimatedJobTime'):
-            jdl.append('+MaxWallTimeMins = %d\n' % int(job['estimatedJobTime'] / 60.0))
+            estimatedMins = int(job['estimatedJobTime'] / 60.0)
+            origCores = int(job.get('numberOfCores', 1))
+            estimatedMinsSingleCore = estimatedMins*origCores
+            jdl.append('+MaxWallTimeMins = WMCore_ResizeJob ? (%d/RequestCpus + 60) : %d\n' % (estimatedMinsSingleCore, estimatedMins))
+            
         if job.get('estimatedMemoryUsage'):
-            jdl.append('request_memory = %d\n' % int(job['estimatedMemoryUsage']))
+            extraMem = int(job.get('extraMemoryPerCore', 500))
```

Comment:
Sure, that's fine.  I would like to see this as configurable per agent - or, rather, in the workflow definition.